python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -53,6 +53,7 @@
 from dagster._core.definitions.tags import build_kind_tag
 from dagster._utils.merger import merge_dicts
 
+from dagster_dbt.asset_specs import AssetSpec
```

Comment:
Looks like this causes a circular import

```suggestion
from dagster import AssetSpec
```